src/saving/SaveList.cs

```diff
@@ -101,6 +104,8 @@ public class SaveList : ScrollContainer
     [Signal]
     public delegate void OnSaveLoaded(string saveName);
 
+    [Signal]
+    public delegate void OnItemsDataLoadedCompleted();
     public override void _Ready()
```

Comment:
There should be a newline between these two elements, that's why Jetbrains is failing